mobile/src/screens/HomeScreen.tsx

```diff
@@ -37,10 +37,11 @@ const TOP_VIDEOS = [
  * Renders the homescreen for the app. Currently displays heading, new books, all books.
  */
 export const HomeScreen: React.FC = () => {
-  // get books from backend
+  // get books and popularBooks from backend
   const booksCtx = useContext(BookContext);
   useEffect(booksCtx.fetchBooks, []);
-  const { books, loading } = booksCtx;
+  useEffect(booksCtx.fetchPopularBooks, []);
+  const { books, popularBooks, loading } = booksCtx;
```

Comment:
just for consistency can you extract `popularLoading` here instead of using `booksCtx.popularLoading` on :137?